python/src/trezorlib/models.py

```diff
@@ -104,9 +112,23 @@ class TrezorModel:
     internal_name="D002",
     minimum_version=(2, 1, 0),
     vendors=VENDORS,
-    usb_ids=((0x1209, 0x53C1), (0x1209, 0x53C0)),
+    usb_ids=(USBID_TREZOR_CORE, USBID_TREZOR_CORE_BOOTLOADER),
+    default_mapping=mapping.DEFAULT_MAPPING,
+)
+
+# ==== unknown model ====
+
+UNKNOWN_MODEL = TrezorModel(
+    name="Unknown Trezor model",
+    internal_name="????",
```

Comment:
Just brainstorming here. Maybe this would be better from the UX perspective and more akin to other models?

```suggestion
    name="Unknown",
    internal_name="UNKN",
```